index.js

```diff
@@ -2335,6 +2353,7 @@
     alt: alt,
     zero: zero,
     reduce: reduce,
+    foldMap: foldMap,
     size: size,
     elem: elem,
     reverse: reverse,
```

Comment:
| Function          | Requirements                          |
| ----------------- | ------------------------------------- |
| `reduce`          | Foldable                              |
| `size`            | Foldable                              |
| `elem`            | Foldable, Setoid                      |
| `foldMap`         | Foldable, Monoid                      |
| `reverse`         | Foldable, Monoid, Applicative         |

This suggests that `foldMap` should be inserted between `elem` and `reverse`. Please rearrange this, the definition, and the tests. :)

Reply:
I think this should be documented as it's not obvious (to me).

Reply:
Good idea. We could define `fantasy-land/lte` and lint this somehow. :D